src/services/residentLife/ResidentStaff.ts

```diff
@@ -28,16 +28,16 @@ type method = {
   PreferredContact: string;
 };
 
-// Fetches the information of an RA from the API endpoint "Housing/ra/{hallId}/{roomNumber}"
+// Fetches the information of an RA from the API endpoint "Housing/ras"
 const fetchRaInfo = (hallId: string, roomNumber: number): Promise<RA[]> =>
-  http.get(`Housing/ra/${hallId}/${roomNumber}`);
+  http.get(`Housing/ras?hallId=${hallId}&roomNumber=${roomNumber}`);
```

Comment:
String interpolation is not the best tool for adding search params to a URL, because there's a lot of characters that have to be specially encoded. Browsers provide the `URLSearchParams` class to make working with search params easier, but in 360 we've made a simple utility function to handle most cases:
```suggestion
  http.get(`Housing/ras${http.toQueryString({hallId, roomNumber})`);
```

Reply:
Updated accordingly